_health-care/_js/reducers/nameAndGeneralInformation/index.js

```diff
@@ -0,0 +1,28 @@
+const initial = {
+  fullName: {
+    first: '',
+    middle: '',
+    last: '',
+    suffix: '',
+  },
+  mothersMaidenName: '',
+  socialSecurityNumber: '',
+  dateOfBirth: {
+    month: 0,
+    day: 0,
+    year: 0,
+  },
+  maritalStatus: null
+};
```

Comment:
I see how this is updating but I would be interested in seeing how we implement passing `maritalStatus` in `AnnualIncomeSection` so that we are not asking questions about spouse income if `maritalStatus` is `null` or `divorced` etc. I think that would give a us a better idea if this is the correct implementation.